src/commands/help.command.ts

```diff
@@ -0,0 +1,50 @@
+import { TransformPipe } from '@discord-nestjs/common';
+
+import { Command, DiscordCommand, UsePipes } from '@discord-nestjs/core';
+import { CommandInteraction } from 'discord.js';
+import { DiscordMessageService } from '../clients/discord/discord.message.service';
+import { GenericCustomReply } from '../models/generic-try-handler';
+
+@Command({
+  name: 'help',
+  description: 'Get help if you&apos;re having problems with this bot',
+})
+@UsePipes(TransformPipe)
+export class HelpCommand implements DiscordCommand {
+  constructor(private readonly discordMessageService: DiscordMessageService) {}
+
+  handler(commandInteraction: CommandInteraction): GenericCustomReply {
+    return {
+      embeds: [
+        this.discordMessageService.buildMessage({
+          title: 'a',
```

Comment:
This seems to be a bad title, right?

Reply:
Yes. See below what happens with the mixing. Perhaps I should re-evaluate this code though